src/kestrel/syntax/kestrel.lark

```diff
@@ -76,11 +76,16 @@ expression: vtrans where_clause? attr_clause? sort_clause? limit_clause? offset_
 // the function in transformer will mal-function in `merge_transformers()`
 vtrans: TRANSFORM "(" VARIABLE ")"
       | VARIABLE
+      | TRANSFORM2 "(" VARIABLE ")"
```

Comment:
I don't think we need TRANSFORM2.  Instead, you can "or" OBSERVED with TIMESTAMPED in TRANSFORM.

Reply:
Hi @pcoccoli , Unfortunately, using or raises error.

Reply:
Hello @pcoccoli , 

As per your request, I made additional changes to have only one rule to describe transformers. However, I had some difficulty with running tests. I would appreciate it if you could run tests here.

Regards